<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1535</width>
    <height>796</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>541</width>
      <height>301</height>
     </rect>
    </property>
    <widget class="QVideoWidget" name="videowidget" native="true">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>480</width>
       <height>272</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/new/prefix1/resource/file(1).jpeg);</string>
     </property>
    </widget>
   </widget>
   <widget class="QTextBrowser" name="info_box">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>660</y>
      <width>471</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QChartView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>30</y>
      <width>381</width>
      <height>331</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="cameraView">
    <property name="geometry">
     <rect>
      <x>49</x>
      <y>370</y>
      <width>480</width>
      <height>240</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/new/prefix1/resource/camera(1).jpeg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>267</x>
      <y>310</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>视频画面</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>257</x>
      <y>620</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>相机画面</string>
    </property>
   </widget>
   <widget class="QLabel" name="resultView">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>30</y>
      <width>480</width>
      <height>272</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/new/prefix1/resource/network.jpg);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>370</y>
      <width>471</width>
      <height>321</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/new/prefix1/resource/cmd.jpeg);</string>
    </property>
   </widget>
   <widget class="QListView" name="listView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1541</width>
      <height>761</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/new/prefix1/resource/bk.png);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="result">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>320</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(193, 193, 193);</string>
    </property>
    <property name="text">
     <string>视频文件</string>
    </property>
   </widget>
   <widget class="QPushButton" name="yolop_process">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>320</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(193, 193, 193);</string>
    </property>
    <property name="text">
     <string>车道线识别</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>817</x>
      <y>310</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>识别画面</string>
    </property>
   </widget>
   <widget class="QTreeView" name="treeView">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>390</y>
      <width>381</width>
      <height>301</height>
     </rect>
    </property>
   </widget>
   <zorder>listView</zorder>
   <zorder>widget_2</zorder>
   <zorder>info_box</zorder>
   <zorder>graphicsView</zorder>
   <zorder>cameraView</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>resultView</zorder>
   <zorder>textBrowser</zorder>
   <zorder>result</zorder>
   <zorder>yolop_process</zorder>
   <zorder>label_3</zorder>
   <zorder>treeView</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1535</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>摄像头</string>
    </property>
    <addaction name="Open"/>
    <addaction name="Stop"/>
    <addaction name="Video"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="Open">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="Stop">
   <property name="text">
    <string>Stop</string>
   </property>
  </action>
  <action name="Select">
   <property name="text">
    <string>Select</string>
   </property>
  </action>
  <action name="Choose">
   <property name="text">
    <string>choose</string>
   </property>
  </action>
  <action name="actionclose">
   <property name="text">
    <string>close</string>
   </property>
  </action>
  <action name="Files">
   <property name="text">
    <string>Files</string>
   </property>
  </action>
  <action name="Video">
   <property name="text">
    <string>Video</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header>qchartview.h</header>
  </customwidget>
  <customwidget>
   <class>QVideoWidget</class>
   <extends>QWidget</extends>
   <header>qvideowidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
